cmd/util/cmd/rollback-executed-height/cmd/root.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	flagDatadir string
+)
+
+// run with `./remove-execution-fork --datadir /var/flow/data/protocol`
+var rootCmd = &cobra.Command{
+	Use:   "remove-execution-result",
+	Short: "remove execution result",
+	Run:   run,
```

Comment:
We should change the command name or the directory name to be consistent (either `remove-execution-result` or `rollback-execution-result`)